dotnet/src/Microsoft.Agents.AI.Workflows/ChatProtocolExecutor.cs

```diff
@@ -40,6 +63,13 @@ protected override RouteBuilder ConfigureRoutes(RouteBuilder routeBuilder)
                            .AddHandler<TurnToken>(this.TakeTurnAsync);
     }
 
+    /// <summary>
+    /// Adds a single chat message to the accumulated messages for the current turn.
+    /// </summary>
+    /// <param name="message">The chat message to add.</param>
+    /// <param name="context">The workflow context in which the executor executes.</param>
+    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
```

Comment:
The default for cancellationToken is clearly visible on the method declaration. Adding this in xmldoc comments is superfluous.